### DISCUSSION TITLE
9.0.0-rc.78 creating a custom module

### DISCUSSION THREAD

**Discussion by ThemeMonkey-Dev** [directus/directus#6434] [Archive]
According to the https://docs.directus.io/guides/modules/ doc I can't create a custom module. When I apply the document, nothing happens.
The icon does not appear in the navigation.
What is a simple custom module example for 9.0.0-rc.78?
src/index.js
src/module.vue
and rollup.config.js

**Comment by nickrum** [comment]
Unfortunately, there are issues currently with custom modules (see issue #6299).
